src/versions/1.7.7/pages/BO/shipping/carriers/index.ts

```diff
@@ -0,0 +1,17 @@
+// Import pages
+import {BOCarriersPageInterface} from '@interfaces/BO/shipping/carriers';
+import {BOCarriersPage as BOCarriersPageVersion} from '@versions/develop/pages/BO/shipping/carriers';
```

Comment:
```suggestion
import {BOCarriersPage as BOCarriersPageVersion} from '@versions/8.2/pages/BO/shipping/carriers';
```

Reply:
We inherit from the next version, not the develop branch (except if develop is next).

Reply:
Done :+1: